### PR TITLE
Onboarding enablers changes

### PR DESCRIPTION
**Pull request by cZikos**
This is an update to documentation for onboarding services due to recent changes.

### CODE REVIEW

File: docs/extend/extend-apiml/api-mediation-onboard-an-existing-java-rest-api-service-without-spring-boot-with-zowe-api-mediation-layer.md [hunk 1]
```diff
@@ -256,6 +256,10 @@ After you add API Mediation Layer integration endpoints, you are ready to add se
     ```yaml
     serviceId: hellospring
     title: HelloWorld Spring REST API
+    eureka:
```
Comment (JirkaAichler) [inline docs/extend/extend-apiml/api-mediation-onboard-an-existing-java-rest-api-service-without-spring-boot-with-zowe-api-mediation-layer.md]: Remove this part. This wasn't added by me and we would need to test it.

File: docs/extend/extend-apiml/api-mediation-onboard-an-existing-java-rest-api-service-without-spring-boot-with-zowe-api-mediation-layer.md [hunk 2]
```diff
@@ -266,13 +270,11 @@ After you add API Mediation Layer integration endpoints, you are ready to add se
     routedServices:
     - gatewayUrl: api/v1
       serviceUrl: /hellospring/api/v1
-
```
Comment (JirkaAichler) [inline docs/extend/extend-apiml/api-mediation-onboard-an-existing-java-rest-api-service-without-spring-boot-with-zowe-api-mediation-layer.md]: In this case we need api-doc endpoint to access swagger. Return back this part.

File: docs/extend/extend-apiml/api-mediation-onboard-an-existing-java-rest-api-service-without-spring-boot-with-zowe-api-mediation-layer.md [hunk 3]
```diff
@@ -312,7 +314,20 @@ After you add API Mediation Layer integration endpoints, you are ready to add se
 
         **Tip:** We recommend that you provide a specific default value of the `title`.
         Use a title that describes the service instance so that the end user knows the specific purpose of the service instance.
+
```
Comment (JirkaAichler) [inline docs/extend/extend-apiml/api-mediation-onboard-an-existing-java-rest-api-service-without-spring-boot-with-zowe-api-mediation-layer.md]: Remove this part. I don't know if it is needed or not.

Comment (cZikos) [reply docs/extend/extend-apiml/api-mediation-onboard-an-existing-java-rest-api-service-without-spring-boot-with-zowe-api-mediation-layer.md]: I am not sure which part you refer to. Title, Description, etc. are different than apiInfo.title, apiInfo.description that were replaced.

File: docs/extend/extend-apiml/api-mediation-onboard-an-existing-java-rest-api-service-without-spring-boot-with-zowe-api-mediation-layer.md (hunk 1 above)
Comment (cZikos) [reply docs/extend/extend-apiml/api-mediation-onboard-an-existing-java-rest-api-service-without-spring-boot-with-zowe-api-mediation-layer.md]: Removed

File: docs/extend/extend-apiml/api-mediation-onboard-an-existing-java-rest-api-service-without-spring-boot-with-zowe-api-mediation-layer.md (hunk 2 above)
Comment (cZikos) [reply docs/extend/extend-apiml/api-mediation-onboard-an-existing-java-rest-api-service-without-spring-boot-with-zowe-api-mediation-layer.md]: Done

File: docs/extend/extend-apiml/api-mediation-onboard-an-existing-java-rest-api-service-without-spring-boot-with-zowe-api-mediation-layer.md [hunk 4]
```diff
@@ -381,18 +382,31 @@ After you add API Mediation Layer integration endpoints, you are ready to add se
...
+        the same as a _gatewayUrl_ value in the _routes_ sections.
+
+    * **apiInfo.swaggerUrl**
+
+        (Optional) Specifies the HTTP or HTTPS address where the Swagger JSON document
+        that provides the API documentation for this API is available.
+        
+    * **apiInfo.documentationUrl**
+
+        (Optional) Link to external documentation, if needed. Can coexist with Swagger documentation.
```
Comment (janan07) [inline docs/extend/extend-apiml/api-mediation-onboard-an-existing-java-rest-api-service-without-spring-boot-with-zowe-api-mediation-layer.md]: Explain 'coexist'. Can this be re-written? 
Example:
"the link to the external documentation can be included along with the Swagger documentation"

File: docs/extend/extend-apiml/api-mediation-onboard-a-spring-boot-rest-api-service.md [hunk 5]
```diff
@@ -388,7 +389,21 @@ As an API service developer, you set multiple configuration settings in your app
...
+      * `eureka.instance.metadata-map.apiml.apiInfo.gatewayUrl`
+      
+          The base path at the API gateway where the API is available. It should bethe same as a _gatewayUrl_ value in the _routes_ sections.
+          
+      * `eureka.instance.metadata-map.apiml.apiInfo.documentationUrl`
+      
+          (Optional) Link to external documentation, if needed. Can coexist with Swagger documentation
+      * `eureka.instance.metadata-map.apiml.apiInfo.swaggerUrl`
+      
+          (Optional) Specifies the HTTP or HTTPS address where the Swagger JSON document that provides the API documentation for this API is available.
```
Comment (janan07) [inline docs/extend/extend-apiml/api-mediation-onboard-a-spring-boot-rest-api-service.md]: This is unclear. Is it clear enough to simply say,
"(Optional) Specifies the HTTP or HTTPS address where the Swagger JSON document is available. "

File: docs/extend/extend-apiml/api-mediation-onboard-a-spring-boot-rest-api-service.md [hunk 6]
```diff
@@ -388,7 +389,21 @@ As an API service developer, you set multiple configuration settings in your app
...
+          Specifies the API identifier that is registered in the API Mediation Layer installation. The API ID uniquely identifies the API in the API Mediation Layer.
+          The same API can be provided by multiple service. The API ID can be used to locate the same APIs that are provided by different services. The creator of the API defines this ID.
+          The API ID needs to be a string of up to 64 characters that uses lowercase alphanumeric characters and a dot: `.`. We recommend that you use your organization as the prefix.
+      * `eureka.instance.metadata-map.apiml.apiInfo.gatewayUrl`
+      
+          The base path at the API gateway where the API is available. Ensure that it is the same path as a _gatewayUrl_ value in the _routes_ sections.
+          
+      * `eureka.instance.metadata-map.apiml.apiInfo.documentationUrl`
+      
+          (Optional) Link to external documentation, if needed. Can coexist with Swagger documentation
```
Comment (janan07) [inline docs/extend/extend-apiml/api-mediation-onboard-a-spring-boot-rest-api-service.md]: Clarify what 'coexisit' means here.

File: docs/extend/extend-apiml/api-mediation-onboard-an-existing-java-rest-api-service-without-spring-boot-with-zowe-api-mediation-layer.md [hunk 7]
```diff
@@ -381,18 +382,31 @@ After you add API Mediation Layer integration endpoints, you are ready to add se
...
+        We recommended that you use your organization as the prefix.
+
+    * **apiInfo.gatewayUrl**
+
+        The base path at the API Gateway where the API is available. Ensure that this is
+        the same path as the _gatewayUrl_ value in the _routes_ sections.
+
+    * **apiInfo.swaggerUrl**
+
+        (Optional) Specifies the HTTP or HTTPS address where the Swagger JSON document
```
Comment (janan07) [inline docs/extend/extend-apiml/api-mediation-onboard-an-existing-java-rest-api-service-without-spring-boot-with-zowe-api-mediation-layer.md]: Lines 403-404 are unclear. Please clarify.

File: docs/extend/extend-apiml/api-mediation-onboard-an-existing-java-rest-api-service-without-spring-boot-with-zowe-api-mediation-layer.md (hunk 4 above)
Comment (cZikos) [reply docs/extend/extend-apiml/api-mediation-onboard-an-existing-java-rest-api-service-without-spring-boot-with-zowe-api-mediation-layer.md]: I replaced it

File: docs/extend/extend-apiml/api-mediation-onboard-a-spring-boot-rest-api-service.md (hunk 5 above)
Comment (cZikos) [reply docs/extend/extend-apiml/api-mediation-onboard-a-spring-boot-rest-api-service.md]: This was a copy from _docs/extend/extend-apiml/api-mediation-onboard-an-existing-rest-api-service-without-code-changes.md. 
I simplified it everywhere in the code.

File: docs/extend/extend-apiml/api-mediation-onboard-a-spring-boot-rest-api-service.md [hunk 8]
```diff
@@ -388,7 +389,21 @@ As an API service developer, you set multiple configuration settings in your app
       * `metadata-map.routed-services.<prefix>.serviceUrl`
 
           Both gateway-url and service-url parameters specify how the API service endpoints are mapped to the API gateway endpoints. The service-url parameter points to the target endpoint on the gateway.
-
+      * `eureka.instance.metadata-map.apiml.apiInfo.apiId`
+      
+          Specifies the API identifier that is registered in the API Mediation Layer installation. The API ID uniquely identifies the API in the API Mediation Layer.
+          The same API can be provided by multiple service. The API ID can be used to locate the same APIs that are provided by different services. The creator of the API defines this ID.
```
Comment (JirkaAichler) [inline docs/extend/extend-apiml/api-mediation-onboard-a-spring-boot-rest-api-service.md]: Please make sure that the parameter description for all enablers is consistent. ' The same API can be provided by multiple service.' should be changed to ' The same API can be provided by multiple serviceS.' There is probably more occurrences.

Comment (cZikos) [reply docs/extend/extend-apiml/api-mediation-onboard-a-spring-boot-rest-api-service.md]: Fixed the "s" for all Docs.

**Comment by cZikos** [comment]
> I fixed 2/4 links. I am having trouble diagnosing 2 of the links, and it will be easier to diagnose once this is merged to master. Feel free to merge.
> 
> FYI @cZikos, I found out that the link you corrected from "gizafoundation", to "zowe.org" was actually a legacy link that is still correct. I reverted that back to https://projectgiza.org/Downloads/post_download.html I don't blame you for trying to fix it though!

Thanks Brandon, I will work on the rest of the links. Yes, my bad, I thought that this was the  link the report was referring to.

File: docs/user-guide/api-mediation/api-mediation-api-catalog.md [hunk 9]
```diff
@@ -6,7 +6,7 @@ As an application developer, use the API Catalog to view what services are runni
 
 * Ensure that the service you would like to view in the API Catalog is configured to be displayed in the API Catalog.
 
-**Note:** For more information about how to configure the API Catalog in the application.yml, see: [Add API Onboarding Configuration](topics/api-mediation/api-mediation-onboard-an-existing-spring-boot-rest-api-service.md).
+**Note:** For more information about how to configure the API Catalog in the application.yml, see: [Add API Onboarding Configuration](topics/api-mediation/api-mediation-onboard-an-existing-sprint-boot-rest-api-service.md).
```
Comment (JirkaAichler) [inline docs/user-guide/api-mediation/api-mediation-api-catalog.md]: Should it be 'spring'?

Comment (janan07) [reply docs/user-guide/api-mediation/api-mediation-api-catalog.md]: Fixed.

**Comment by cZikos** [comment]
This PR is by mistake to be merged to master. PR #418 was created instead, to be merged to docs-staging.

**Pull request closed without merging**